service/kvmconfig/v8/version_bundle.go

```diff
@@ -9,39 +9,9 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
-			{
-				Component:   "kvm-node-controller",
-				Description: "Updated KVM node controller with pod status bugfix.",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "Calico",
-				Description: "Updated to 3.0.2.",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "kubelet",
-				Description: "Tune kubelet flags for protecting key units (kubelet and container runtime) from workload overloads.",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "etcd",
-				Description: "Updated to 3.3.1.",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "qemu",
-				Description: "Fixed formula for calculating qemu memory overhead.",
-				Kind:        versionbundle.KindFixed,
-			},
-			{
-				Component:   "monitoring",
-				Description: "Added configuration for monitoring endpoint IP addresses.",
-				Kind:        versionbundle.KindAdded,
-			},
 			{
 				Component:   "cloudconfig",
-				Description: "Enable aggregation layer to be able to extend kubernetes API.",
+				Description: "Removed set-ownership-etcd-data-dir.service.",
```

Comment:
Do you know this already changed?

Reply:
it will be there because of my next change, but I agree it is better to keep the history, so I'm updating this with placeholder